cypress/integration/a11y_spec.js

```diff
@@ -4,40 +4,6 @@ describe('CheckA11y.css Tests', () => {
   const WARNING_BOXSHADOW = 'rgb(255, 255, 102) 0px 0px 0px 4px';
   const ERROR_BORDER = '6.4px solid rgb(255, 0, 0)';
```

Comment:
I think that would be a good idea. Strangely two show as unused only when I edit the file... But unlike the commands I move I don't know if these should live there or have a strong feeling about where they should move. Happy to follow direction, but otherwise I'd like to squeak that.

Reply:
Also keep in mind commands were including this file full of comments before. We have no such file for constants.

Reply:
@alvaromontoro Are you happy with this? 😊

This is awesome work you two! Thank you!!

Reply:
NP jack. @alvaromontoro if not fine, let me know where you would like the files located for the constants so I'm not left guessing.

Reply:
I'm always happy with everything. Actually, I approved the changes yesterday :P

Reply:
Yeah, sorry, should have more clear. I'm very unfamiliar with Cypress and haven't had chance to fully check it out (100% of my time currently is on another project, which should finish on Saturday). I noticed you left a comment and wanted to make sure you didn't think an action was needed.

Thank you both! @Lewiscowles1986 I will merge your PR now and follow your instructions. 💪